src/services/sync_mutex.ts

```diff
@@ -6,7 +6,7 @@
 const Mutex = require('async-mutex').Mutex;
 const instance = new Mutex();
 
-async function doExclusively(func) {
+async function doExclusively<T>(func: () => void) {
```

Comment:
Does func not return anything? Still ramping up but line 13 appears to return a resolved function, why return anything here if the function returns void?

Reply:
Guessing maybe the generic was intended here since I am not seeing it used elsewhere?

Reply:
@j-tirelli , you're absolutely right. I've checked if `doExclusively`'s return value is used and it is (in `backup.js`). I've changed `void` to `T` and everything is fine.